toolset/databases/mysql/mysql.dockerfile

```diff
@@ -1,44 +1,11 @@
-FROM buildpack-deps:bionic
+FROM mysql:8
```

Comment:
If this does work, let's use the full patch version for reproducible results

Reply:
I can make it 8.0.18 but 8 is running 8.0.18. When you use 8 it will keep up with the patches each time they release.

Reply:
Each commit in this repo needs to have reproducible results. If at some point, you want to rerun results for Round 18, the MySQL version used during the run and the version used when trying to reproduce results would be different. As much as possible we want to lock down specific versions. This is true across framework dependencies as well.